applications/convection_diffusion_application/python_scripts/python_solvers_wrapper_convection_diffusion.py

```diff
@@ -45,7 +45,7 @@ def CreateSolverByParameters(model, solver_settings, parallelism):
         err_msg += "Available options are: \"OpenMP\", \"MPI\""
         raise Exception(err_msg)
 
-    module_full = 'KratosMultiphysics.convection_diffusion_application.' + solver_module_name
+    module_full = 'KratosMultiphysics.ConvectionDiffusionApplication.' + solver_module_name
```

Comment:
my bad, sorry
We should probably change the name of the app, then those problems would be solved for good :)